setup/schemas.sql

```diff
@@ -0,0 +1,77 @@
+USE library_management_system;
+
+IF NOT EXISTS
+    (
+        SELECT TOP 1 1
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_SCHEMA = 'dbo'
+            AND TABLE_NAME = 'book'
+            AND COLUMN_NAME = 'book_id'
+    )
+BEGIN
+    CREATE TABLE book (
+    book_id INT PRIMARY KEY IDENTITY (1, 1),
+    title NVARCHAR(100) NOT NULL,
+    author NVARCHAR(100) NOT NULL,
+    isbn NVARCHAR(17) NOT NULL,
```

Comment:
make it unique, isbn is unique for each book.

also check out isbn format and adjust its length according to it

Reply:
IBSN is unique for each edition, not for each book. And since any library can easily have more than one copy of the same edition of the book, the unique constraint won't work here.

I also checked the  ISBN format, there are two versions of the IBSON format, ISBN 10 and ISBN 13, and it contains 17 characters at most if we count the hyphens too.